ni_measurement_plugin_sdk/discovery/_support.py

```diff
@@ -57,7 +57,7 @@ def _get_discovery_service_location() -> pathlib.PurePath:
 
 def _get_registration_json_file_path() -> pathlib.Path:
     if sys.platform == "win32":
-        return _get_nipath("NISHAREDDIR64") / "MeasurementLink" / "MeasurementLinkServices.json"
+        return _get_nipath("NISHAREDDIR64") / "Services" / "Services.json"
```

Comment:
After discussing with Ryan Eckenrode, we decided to change this to `NIServices.json` since searching the ancestor directory hierarchy for Services.json from the C# client is too generic and might pick up some JSON file we don't own.

Reply:
Updated to `NIServices.json`.